torch/ao/quantization/experimental/quantizer.py

```diff
@@ -47,28 +33,41 @@ def __init__(
     def quantize_APoT(self, tensor2quantize: Tensor):
         result = torch.tensor([])
         # map float_to_apot over tensor2quantize elements
-        result = tensor2quantize.apply_(lambda x: float_to_apot(x, self.quantization_levels, self.level_indices))
+        result_data = tensor2quantize.apply_(lambda x: float_to_apot(x, self.quantization_levels, self.level_indices))
+
+        from torch.ao.quantization.experimental.APoT_tensor import TensorAPoT
+
+        result = TensorAPoT(self, result_data)
 
         return result
 
-    r""" Dequantizes integer Tensor to floating point representation
+    r""" Dequantizes integer Tensor to floating point (fp) representation
     based on the calculated quantization levels from a specified APoT non-uniform observer.
     The approach follows the method outlined in the APoT paper: https://arxiv.org/pdf/1909.13144.pdf.
     Args:
-        self: APoTQuantizer with attr data to dequantize
+        float2apot: quantized APoT Tensor to dequantize
     Returns:
-        result: floating point representation of input Tensor
+        result: fp representation of input Tensor
     """
-    def dequantize(self, float2apot: Tensor):  # type: ignore[override]
-        float2apot = float2apot.float()
-
-        quantization_levels = self.quantization_levels
-        level_indices = self.level_indices
+    def dequantize_APoT(self, float2apot):
+        apot_tensor = float2apot.data
 
         # map apot_to_float over tensor2quantize elements
-        result = float2apot.apply_(lambda x: float(apot_to_float(x, quantization_levels, level_indices)))
+        result = apot_tensor.apply_(lambda x: float(apot_to_float(x, self.quantization_levels, self.level_indices)))
 
         return result
 
     def q_apot_alpha(self) -> float:
         raise NotImplementedError
+
+r""" Global method to create quantizer and call quantizer quantize_APoT
+"""
+def quantize_APoT(tensor2quantize: Tensor, qparams: Tuple):
+    quantizer = APoTQuantizer(alpha=qparams[0], gamma=qparams[1], quantization_levels=qparams[2], level_indices=qparams[3])
+    return quantizer.quantize_APoT(tensor2quantize)
+
+r""" Global method to create quantizer and call quantizer dequantize_APoT
+"""
+def dequantize_APoT(float2apot, qparams: Tuple):
+    quantizer = APoTQuantizer(alpha=qparams[0], gamma=qparams[1], quantization_levels=qparams[2], level_indices=qparams[3])
+    return quantizer.dequantize_APoT(float2apot)
```

Comment:
`float2apot` --> `apot_tensor`?

Reply:
also it can be:
```
def deqauntize_APoT(apot_tensor):
        return apot_tensor.quantizer.dequantize(apot_tensor)
```
please add type annotations as well